Makefile

```diff
@@ -58,9 +58,24 @@ help:			## Show this help.
 clean:			## Remove all build and test related artifacts
 	rm -rf ./.out bin
 
-test: $(KUBEBUILDER_ASSETS)          ## Run local tests
+test-local: $(KUBEBUILDER_ASSETS)          ## Run local tests
```

Comment:
TODO: job calls `make test` which is why I took over the test target and set this to `test-local`. Do we want to leave this as test, make the other target `test-ci`, and change the job to run that?

At some point we may also want to split out unit from integration but I haven't done that yet. At that point we may want to consider bash instead of me doing horrible things to the makefile :)